priam/src/main/java/com/netflix/priam/scheduler/SchedulerType.java

```diff
@@ -0,0 +1,31 @@
+package com.netflix.priam.scheduler;
+
+/**
+ * Created by aagrawal on 3/8/17.
+ */
+public enum SchedulerType {
+    TIME("TIME"),CRON("CRON");
+
+    private final String schedulerType;
+
+    SchedulerType(String schedulerType)
+    {
+        this.schedulerType = schedulerType.toUpperCase();
+    }
+
+    public String getSchedulerType()
+    {
+        return schedulerType;
+    }
+
+    //Helper method to find the scheduler type - case insensitive as user may
+    //put value which are not right case.
+    public static SchedulerType valueOfIgnoreCase(String schedulerType) {
+        return SchedulerType.valueOf(schedulerType.toUpperCase());
+    }
+
+    public static SchedulerType lookup(String schedulerType) {
+        return valueOfIgnoreCase(schedulerType);
```

Comment:
This might throw a runtime exception if the input is not in one of the provided values. Something like below would be safer instead of throwing exceptions in the main code path. Throwing exception might work in current scenario but might for any other future users of this enum.

```
public static Blah fromString(String text) {
    for (Blah b : Blah.values()) {
      if (b.text.equalsIgnoreCase(text)) {
        return b;
      }
    }
    return null;
  }
```